packages/ics721/src/state.rs

```diff
@@ -19,9 +19,15 @@ pub const PO: PauseOrchestrator = PauseOrchestrator::new("c", "d");
 
 /// Maps classID (from NonFungibleTokenPacketData) to the cw721
 /// contract we have instantiated for that classID.
-pub const CLASS_ID_TO_NFT_CONTRACT: Map<ClassId, Addr> = Map::new("e");
-/// Maps cw721 contracts to the classID they were instantiated for.
-pub const NFT_CONTRACT_TO_CLASS_ID: Map<Addr, ClassId> = Map::new("f");
+/// NOTE: legacy stores with keys `e` and `f` are no longer used.
+pub const CLASS_ID_AND_NFT_CONTRACT_INFO: IndexedMap<&str, ClassIdInfo, ClassIdInfoIndexes> =
+    IndexedMap::new(
+        "m",
+        ClassIdInfoIndexes {
+            class_id: UniqueIndex::new(|d| d.class_id.clone(), "class_id_info__class_id"),
+            address: UniqueIndex::new(|d| d.address.clone(), "class_id_info__address"),
```

Comment:
@humanalgorithm IndexedMap allows multiple indices. Here we need to put an index on `ClassId` and collection address.